src/measure_theory/decomposition/signed_hahn.lean

```diff
@@ -358,7 +358,7 @@ lemma bdd_below_measure_of_negatives :
   bdd_below s.measure_of_negatives :=
 begin
   simp_rw [bdd_below, set.nonempty, mem_lower_bounds],
-  by_contra, push_neg at h,
+  by_contra',
```

Comment:
The name `h` is used later on, so it should be introduced explicitly. The original proof was "bad" in this respect.
```suggestion
  by_contra' h,
```